src/mode/modeHandler.ts

```diff
@@ -33,6 +33,11 @@ export enum VimSpecialCommands {
   Dot
 }
 
+export class ViewChange {
+  public name: string;
```

Comment:
nit: maybe rename this to `command`? As that'll be inline with how it's used in `executeCommand`.